tests/utils/test_config.py

```
@@ -0,0 +1,120 @@
+"""Tests for configuration utilities."""
+import os
+import pytest
+from pathlib import Path
+from unittest.mock import patch, Mock
+
+
+@pytest.mark.unit
+class TestConfig:
+    """Test configuration utilities."""
+
+    def test_validate_config_success(self, mock_env_vars, tmp_path):
```

Ruff (0.14.5)

Unused method argument: `mock_env_vars`

(ARG002)